build.gradle

```diff
@@ -99,8 +99,8 @@ subprojects { subproject ->
 		log4j2Version = '2.7'
 		logbackVersion = '1.1.7'
 		mockitoVersion = '1.10.19'
-		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '3.6.5'
-		rabbitmqHttpClientVersion = '1.0.0.RELEASE'
+		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '4.0.0'
```

Comment:
Eh... Yes, since client 4.0 is compatible with Broker 3.6 and this amqp-client is transitive dependency of Spring AMQP and I'm pretty sure there is no any other Spring project depended of amqp-client, therefore we are safe to upgrade to this major version.
Yes, it breaks a bit our IO Platform rules, but still arguments to upgrade and forget are valuable enough.
WDYT, @wilkinsona ?